src/dalle_mini/model/modeling.py

```diff
@@ -561,3 +563,24 @@ def _decoder_forward(
             outputs = outputs[:1] + (unfreeze(past["cache"]),) + outputs[1:]
 
         return outputs
+
+    @classmethod
+    def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
+        """
+        Initializes from a wandb artifact, or delegates loading to the superclass.
+        """
+        if ":" in pretrained_model_name_or_path and not os.path.isdir(
+            pretrained_model_name_or_path
+        ):
+            # wandb artifact
+            artifact = wandb.Api().artifact(pretrained_model_name_or_path)
+
+            # we download everything, including opt_state, so we can resume training if needed
+            # see also: #120
```

Comment:
probably can remove this line (otherwise we may forget)

Reply:
Do you mean the "see also"? Sure, no problem. In fact, if we do #120 we don't need the other comment either.

Reply:
Yeah so my logic is to add a note only if it's not an issue/PR linked to our repo as there are no ways to have a reference. If it's very important then we can also track the issue in our repo instead.